technical_standards/testing/index.md

```diff
@@ -0,0 +1,38 @@
+# Testing
+
+While automated testing is often praised, it is important to remember that they are just a tool with their pros and cons.
+
+That is why it is important to get them right to not end up with a burden slowing your developing workflow.
+
+## Testing everything you should not
+
+Automating a test is not mandatory, and there are cases there you should not automate them.
+
+To understand this, you should consider the time you will spend on creating and maintaining the test as an investment.
+
+All that will make the difference between automated and manual testing is if the time you are going to save from the test running will be higher from the time you invested.
+
+That way, achieving high coverage is still an objective, but achieving 100% coverage is in fact a way to detect something wrong in the practices.
```

Comment:
As part of a handbook, guidelines should be actionble. As a new comer, I am not sure I'd be able to know what to test/what not reading this. Maybe some examples would be beneficial here?

Reply:
> For the second part, would it be too much pointing finger?

To avoid pointing fingers, find positive examples, not negative ones (ie. examples in which the guideline is correctly followed). You can probably find in the codebase or build examples of code that should not necessarily be covered by tests, that is actually not covered by test.